SpatialGDK/Extras/schema/spawner.schema

```diff
@@ -6,6 +6,7 @@ type SpawnPlayerRequest {
     bytes unique_id = 2;
     string online_platform_name = 3;
     bool simulated = 4;
+    string client_worker_id = 5;
```

Comment:
Can we move this field to the forwarded player spawn request ?
I think we should get this field's value on the server from the Worker API when receiving the original spawn request from the client, and only add it when communicating between servers.

Reply:
can od sure